packages/shared/src/utils/os/os_detect.ts

```diff
@@ -42,6 +42,10 @@ export const isDesktopOs = () => {
 export const isMobileOs = () =>
     /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent);
 
+export const isTabletOs =
+    /ipad|android 3.0|xoom|sch-i800|playbook|tablet|kindle/i.test(navigator.userAgent.toLowerCase()) ||
+    (/android/i.test(navigator.userAgent) && !/mobile/i.test(navigator.userAgent));
```

Comment:
If you added `navigator.userAgent.toLowerCase()` for checking in line 46, should you also add it here?

Reply:
I forgot to make it lowercase. And for the second question, android 3.0 is os for large mobile device or tablet. In case both are true means, it is a tablet